src/manager.js

```diff
@@ -15,6 +16,7 @@ var AdManager = function(options) {
     debug: false,
     dfpId: 4246,
     amazonEnabled: true,
+    pbjsEnabled: true,
```

Comment:
Just to be on the safe side, this should have a default value of `false` for now